site/docs/components/static-list/examples.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+
+  <LivePreview componentName="static-list" exampleName="Default" >
+
+## Default
+
+The default `Static list` allows users to show non-interactive list items.
+
+  </LivePreview>
+
+  <LivePreview componentName="static-list" exampleName="ComplexLabel" >
+
+## Complex label
+
+For complex labels, you can add additional elements to the static content inside `StaticListItemContent` to provide further context or descriptions for each list item.
```

Comment:
List content component seems still a little hard to justify with some examples with only one text in them

Reply:
removed the wrapper for the simple example and moved this to a note that says you can wrap if you need to ensure spacing is correct when using icons and buttons